<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>980</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ViennaCLBench - A free open-source benchmark</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/otherFiles.qrc">
    <normaloff>:/resources/images/benchmarkIcon.ico</normaloff>:/resources/images/benchmarkIcon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="mainGridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="MenuListWidget" name="mainMenuListWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QListWidget 
{ 
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(41, 41, 41, 255), stop:1 rgba(85, 85, 85, 255));  
}

QListWidget::item:hover 
{ 
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(150, 150, 150, 255), stop:1 rgba(200, 200, 200, 255));
}

QListWidget::item:selected 
{ 
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(200, 200, 200, 200), stop:1 rgba(220, 220, 220, 255)); 
border-top: 2px groove gray; 
color: #404040;
}
QIcon
{
background: red;selection-color: red; 
}


QIcon::selected
{
background: red;selection-color: red; 
}

QListWidget::item 
{ 
padding: 10px; color: white; 
border-bottom: 2px groove gray; 
}</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="autoScroll">
         <bool>false</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::NoDragDrop</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="movement">
         <enum>QListView::Static</enum>
        </property>
        <property name="flow">
         <enum>QListView::TopToBottom</enum>
        </property>
        <property name="isWrapping" stdset="0">
         <bool>true</bool>
        </property>
        <property name="layoutMode">
         <enum>QListView::SinglePass</enum>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
        <property name="selectionRectVisible">
         <bool>true</bool>
        </property>
        <property name="currentRow">
         <number>-1</number>
        </property>
        <item>
         <property name="text">
          <string>Home</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignTop</set>
         </property>
         <property name="icon">
          <iconset resource="../../resources/icons.qrc">
           <normaloff>:/icons/icons/home_w.png</normaloff>:/icons/icons/home_w.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Benchmark</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
         <property name="icon">
          <iconset resource="../../resources/icons.qrc">
           <normaloff>:/icons/icons/bar_chart.png</normaloff>:/icons/icons/bar_chart.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>My Results</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
         <property name="icon">
          <iconset resource="../../resources/icons.qrc">
           <normaloff>:/icons/icons/benchmarkIcon2.png</normaloff>:/icons/icons/benchmarkIcon2.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Database</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
         <property name="icon">
          <iconset resource="../../resources/icons.qrc">
           <normaloff>:/icons/icons/db.png</normaloff>:/icons/icons/db.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MatrixMarket</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
         <property name="icon">
          <iconset resource="../../resources/icons.qrc">
           <normaloff>:/icons/icons/matrix.gif</normaloff>:/icons/icons/matrix.gif</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MatrixMarket2</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
         <property name="icon">
          <iconset resource="../../resources/icons.qrc">
           <normaloff>:/icons/icons/matrix.gif</normaloff>:/icons/icons/matrix.gif</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>System Info</string>
         </property>
         <property name="textAlignment">
          <set>AlignHCenter|AlignVCenter|AlignCenter</set>
         </property>
         <property name="icon">
          <iconset resource="../../resources/icons.qrc">
           <normaloff>:/icons/icons/systemInfo.png</normaloff>:/icons/icons/systemInfo.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>5</number>
        </property>
        <widget class="QWidget" name="homePage">
         <property name="styleSheet">
          <string notr="true">QWidget#homePage
{ 
background: #F0F0F0; 
}</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="HomeScreen" name="homeScreen" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="benchmarkPage">
         <property name="styleSheet">
          <string notr="true">QWidget#benchmarkPage
{ 
background:#F0F0F0; 
}
</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QTabWidget" name="benchmarkPageTabWidget">
            <property name="styleSheet">
             <string notr="true">QTabBar::tab{
width: 250px;
height: 35px;
/*
background: gray;*//*
border-bottom: 1px groove gray;*/
}
QTabWidget::pane {

 border: 1px solid #C4C4C3;
top: -1px; 
}
QTabBar::tab:selected{
border-left: 1px solid gray;
border-right: 1px solid gray;
border-top: 1px solid gray;
border-bottom: 1px solid #F0F0F0;
background: #F0F0F0;/*
border-bottom: 1px groove #F0F0F0;
border-left: 1px groove gray;
border-right: 1px groove gray;
border-top: 1px groove gray;*/

}</string>
            </property>
            <property name="tabShape">
             <enum>QTabWidget::Rounded</enum>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="basic_BenchmarkTab">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QTabBar::tab{
/*
background: gray;*//*
border-bottom: 1px groove gray;*/
}
QTabWidget::pane {

}
QTabBar::tab:selected{

}</string>
             </property>
             <attribute name="title">
              <string>Standard</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="BenchmarkScreen" name="basicBenchmark" native="true"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="expert_BenchmarkTab">
             <attribute name="title">
              <string>Advanced</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="ExpertBenchmarkScreen" name="expertBenchmark" native="true"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="myResultsPage">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="ResultHistory" name="resultHistory" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="databasePage">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="databaseLabel">
            <property name="text">
             <string>Online Result Database</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="databaseTable">
            <row>
             <property name="text">
              <string>1.</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2.</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Date</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Result</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Note</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>23.06.2014</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>7560</string>
             </property>
            </item>
            <item row="0" column="2">
             <property name="text">
              <string>Test1</string>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string>24.06.2014</string>
             </property>
            </item>
            <item row="1" column="1">
             <property name="text">
              <string>7850</string>
             </property>
            </item>
            <item row="1" column="2">
             <property name="text">
              <string>Test2</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="matrixMarketPage">
         <layout class="QGridLayout" name="gridLayout_8">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="MatrixMarket_Widget" name="matrixMarket_Widget" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="matrixMarket_AlternativePage">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="MatrixMarket_Alternative" name="matrixMarket_Alternative" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="systemInfoPage">
         <layout class="QGridLayout" name="gridLayout_7" rowstretch="0" columnstretch="0">
          <item row="0" column="0">
           <widget class="SystemInfoScreen" name="widget" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar{
	background: qlineargradient(x1:0,x2:0,y1:0,y2:1,stop:0 #525252, stop:0.4 #323232);
	color: white;
}
QMenuBar:item{
	background: qlineargradient(x1:0,x2:0,y1:0,y2:1,stop:0 #525252, stop:0.4 #323232);
	color: white;
}
QMenuBar:item:selected{
	background: qlineargradient(x1:0,x2:0,y1:0,y2:1,stop:0 #787878, stop:0.4 #D8D8D8);
	color: black;
}</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MenuListWidget</class>
   <extends>QListWidget</extends>
   <header>src/ui/widgets/menulistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>MatrixMarket_Widget</class>
   <extends>QWidget</extends>
   <header>src/ui/matrixmarket_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HomeScreen</class>
   <extends>QWidget</extends>
   <header>src/ui/homescreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SystemInfoScreen</class>
   <extends>QWidget</extends>
   <header>src/ui/systeminfoscreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ExpertBenchmarkScreen</class>
   <extends>QWidget</extends>
   <header>src/ui/expertbenchmarkscreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ResultHistory</class>
   <extends>QWidget</extends>
   <header>src/ui/resulthistory.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BenchmarkScreen</class>
   <extends>QWidget</extends>
   <header>src/ui/benchmarkscreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MatrixMarket_Alternative</class>
   <extends>QWidget</extends>
   <header>src/ui/matrixmarket_alternative.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/otherFiles.qrc"/>
  <include location="../../resources/icons.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>startBenchmark()</slot>
  <slot>updateToggleButtons()</slot>
 </slots>
</ui>
